src/main/webapp/app/exam/manage/exams/exam-detail.component.html

```diff
@@ -108,163 +108,7 @@ <h2 id="exam-detail-title">
-                    <dd>
-                        @if (exam.numberOfCorrectionRoundsInExam) {
-                            <span>{{ exam.numberOfCorrectionRoundsInExam }}</span>
-                        } @else {
-                            <span jhiTranslate="artemisApp.exam.notSet"></span>
-                        }
-                    </dd>
-                    <dt><span jhiTranslate="artemisApp.examManagement.randomizeQuestionOrder">Randomize order of exercise groups</span></dt>
-                    <dd>
-                        <span>{{ exam.randomizeExerciseOrder }}</span>
-                    </dd>
-                    <dt><span jhiTranslate="artemisApp.examManagement.examStudents.registeredStudents">Registered students</span></dt>
-                    <dd>
-                        @if (exam.numberOfExamUsers) {
-                            <span>{{ exam.numberOfExamUsers }}</span>
-                        } @else {
-                            {{ 0 }}
-                        }
-                    </dd>
-                    <dt><span jhiTranslate="artemisApp.examManagement.startText">Exam start text</span></dt>
-                    @if (formattedStartText) {
-                        <dd id="exam-start-text" class="editor-outline-background" [innerHTML]="formattedStartText"></dd>
-                    } @else {
-                        <span jhiTranslate="artemisApp.exam.notSet"></span>
-                    }
-                    <dt><span jhiTranslate="artemisApp.examManagement.confirmationStartText">Exam start confirmation text</span></dt>
-                    @if (formattedConfirmationStartText) {
-                        <dd id="exam-confirmation-start-text" class="editor-outline-background" [innerHTML]="formattedConfirmationStartText"></dd>
-                    } @else {
-                        <span jhiTranslate="artemisApp.exam.notSet"></span>
-                    }
-                    <dt><span jhiTranslate="artemisApp.examManagement.endText">Exam end text</span></dt>
-                    @if (formattedEndText) {
-                        <dd id="exam-end-text" class="editor-outline-background" [innerHTML]="formattedEndText"></dd>
-                    } @else {
-                        <span jhiTranslate="artemisApp.exam.notSet"></span>
-                    }
-                    <dt><span jhiTranslate="artemisApp.examManagement.confirmationEndText">Exam end confirmation text</span></dt>
-                    @if (formattedConfirmationEndText) {
-                        <dd id="exam-confirmation-end-text" class="editor-outline-background" [innerHTML]="formattedConfirmationEndText"></dd>
-                    } @else {
-                        <span jhiTranslate="artemisApp.exam.notSet"></span>
-                    }
-                </dl>
+                <jhi-detail-overview-list [sections]="examDetailSections"></jhi-detail-overview-list>
```

Comment:
Do you need to check for examDetailSections, don't you? (@if (examDetailSections))